al_log_filter.js

```diff
@@ -7,16 +7,18 @@
  * @end
  * -----------------------------------------------------------------------------
  */
-const where = require('lodash.where');
+const lodashFilter = require('lodash.filter');
+const lodashRemove = require('lodash.remove');
+const lodashcloneDeep = require('lodash.clonedeep');
 
 /**
  *  @function initializes JSON filter
- *  
+ *
  *  @param filter - string or object, for example, '{"a": 1}', {a:1}
```

Comment:
Is it possible to use array as input parameter?
could you please also add more filter examples here.

Reply:
Done